README.md

```diff
@@ -35,7 +35,9 @@ in GitHub.
 
 ## ChangeLog
 
-* issue#41:  Maintenance schedule check add
+* issue#42: Search string fix
```

Comment:
Newer issue should come below older issue

Reply:
It is changed.